model/lib/oidc/token_id.dart

```diff
@@ -8,4 +8,5 @@ class TokenId with EquatableMixin {
 
   @override
   List<Object> get props => [uuid];
+
```

Comment:
Why did you fix it like that?

Reply:
no, temporary change to trigger CICD :), please help me remove it